tools/export/iar/iar_definitions.json

```diff
@@ -371,7 +371,7 @@
     "STM32H743ZI": {
         "OGChipSelectEditMenu": "STM32H743ZI\tST STM32H743ZI"
     },
-    "DISCO_H747I": {
+    "STM32H747XIHx": {
         "OGChipSelectEditMenu": "STM32H747XI_CM7\tST STM32H747XI_CM7"
     },
     "DISCO_H747I_CM4": {
```

Comment:
And in case the change is right, shouldn't the CM4 reference be updated as well ?

Reply:
No: in fact both CM7 and CM4 share the same device name STM32H747XIHx, but the same name can't be used to export both. So I removed "device_name" for DISCO_H747I_CM4 in targets.json. ("device_name" was required only for CM7)